redis.c

```diff
@@ -1982,6 +1982,8 @@ PHP_METHOD(Redis, exec)
         RETURN_FALSE;
     }
 
+    ZVAL_NULL(&z_ret);
```

Comment:
You should use `ZVAL_FALSE` here because with `NULL` there is a return type mismatch error
`PHP Fatal error:  Redis::exec(): Return value must be of type Redis|array|false, null returned in Unknown on line `0